storage/backend/ceph/util.go

```diff
@@ -13,7 +14,13 @@ func (c *Driver) MountPath(do storage.DriverOptions) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return filepath.Join(c.mountpath, do.Volume.Params["pool"], volName), nil
+
+	poolName, ok := do.Volume.Params["pool"].(string)
+	if !ok {
+		return "", errored.Errorf("Expected string type for driver.pool")
```

Comment:
in fact, I think pulling out the string should just probably be its own function; this way the access is normalized and we can be a little safer that way. wdyt?